LandSandBoat/server
# Orobon family

**_I affirm:_**

## What does this pull request do?

These changes focus on the Orobon family. This does the following:

- Adds all of the missing Orobon mobskills except for Mayhem Lantern which is from Abyssea era
- Implements mob family logic for breaking the eyestalks and getting Orobon Lure as loot

## Retail Proof

[Orobon Wiki](https://ffxiclopedia.fandom.com/wiki/Category:Orobon)
[Jimmayus' Mob Spreadsheet](https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit?gid=57955395#gid=57955395)
[Giant Orobon Capture 1](https://www.youtube.com/watch?v=3AiUOr3cRWE)
[Giant Orobon Capture 2](https://www.youtube.com/watch?v=YvJNyHEbzqo)
[Angler Orobon (ISNM) Capture](https://www.youtube.com/watch?v=D-aiHPGJRYc)

## Steps to test these changes

`!zone Arrapago Reef`
`!spawnmob 16998406`
`!mobhere 16998406`

Engage the Orobon and use the following:
`!exec target:useMobAbility(1693)`
`!exec target:useMobAbility(1694)`
`!exec target:useMobAbility(1695)`
`!exec target:useMobAbility(1696)`
`!exec target:useMobAbility(1697)`
`!exec target:useMobAbility(1698)`
`!exec target:useMobAbility(1977)`
`!exec target:useMobAbility(1978)`

With `!tp 3000` you can test to make sure it only uses Hypnic Lamp while it has its eyestalks. Destroy the eyestalks by doing critical hits. Orobon Lure should only drop as loot when the eyestalks are destroyed.
